spec/helpers/budget_warning_spec.rb

```diff
@@ -21,17 +21,8 @@
     end
   end
 
-
-  describe '#whitelisted pages' do
-    context 'english page for tools' do
-      ['redundancy-pay-calculator',
-        'workplace-pension-contribution-calculator',
-        'pension-calculator'].each do |tool_name|
-            it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
-               let(:url) { "www.example.com/en/tools/#{tool_name}/" }
-            end
-      end
-    end
+  it_behaves_like 'displays_banner_warning_for_whitelisted_tools' do
```

Comment:
You've removed the `describe` and the `context` blocks here. I think those are useful, how about merging it with the `describe` block below (line 28) and adding the same context that used to be here: "english page for tools"?